api/users/migrations/0001_initial.py

```diff
@@ -15,6 +15,9 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('auth', '0008_alter_user_username_max_length'),
+        # ensure Axes migrations are run before all other migrations to avoid
+        # an error found when running the below migration after upgrading to 5.3.0
+        ('axes', '0007_alter_accessattempt_unique_together'),
```

Comment:
This was an odd one. I was seeing the following error when running the tests at the point it was trying to run the above migration. 

```
ValueError: The field features.FeatureState.identity was declared with a lazy reference to 'environments.identity', but app 'environments' doesn't provide model 'identity'.
```

By adding this dependency here, we make sure that the axes migrations are all run before any of our migrations and hence negate the issues I saw here. It doesn't mean I understand why they happened and I'm pretty sure we'll see it in the future but hopefully this will serve as an example on how to 'fix' it.

Reply:
> 

I did look into a bit, and it seems like it's not using `SeparateDatabaseAndState` correctly and still thinks that `Identity` lives in `environment.models` not sure why tho? so yeah not much useful